packages/blade/src/components/Breadcrumb/_decisions/decisions.md

```diff
@@ -0,0 +1,203 @@
+# Breadcrumb Decisions <!-- omit in toc -->
+
+Breadcrumbs are used for navigating through or to show user’s location in an application.
+
+<img width="426" alt="Breadcrumb Thumbnail" src="./breadcrumb-thumbnail.png" />
+
+- [Design](#design)
+- [Anatomy](#anatomy)
+- [Basic Usage](#basic-usage)
+- [API](#api)
+  - [Breadcrumb](#breadcrumb)
+  - [BreadcrumbItem](#breadcrumbitem)
+  - [Examples](#examples)
+    - [Show Last Separator](#show-last-separator)
+  - [Usage with Routing Libraries](#usage-with-routing-libraries)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade-DSL?type=design&node-id=81170-57117&mode=design&t=qeUPSKFBYZfBCFBT-0) to all variants of the Breadcrumb component
+
+## Anatomy
+
+<img src="./breadcrumb-anatomy.png" width="100%" alt="Anatomy of breadcrumbs" />
+
+## Basic Usage
+
+```jsx
+<Breadcrumb size="small" color="primary">
+  <BreadcrumbItem href="/">Home</BreadcrumbItem>
+  <BreadcrumbItem href="/dashboard">Dashboard</BreadcrumbItem>
+  <BreadcrumbItem href="/settlements" isCurrentPage>Settlements</BreadcrumbItem>
+</Breadcrumb>
+```
+
+## API
+
+### Breadcrumb
+
+```ts
+type BreadcrumbProps = StyledPropsBlade & {
+  /**
+   * Size of the Breadcrumb
+   * 
+   * @default medium
+   */
+  size?: 'small' | 'medium' | 'large';
+  /**
+   * Color of the Breadcrumb
+   * 
+   * @default neutral
+   */
+  color?: 'neutral' | 'primary' | 'white';
+  /**
+   * Content of the Breadcrumb, accepts BreadcrumbItem
+   */
+  children: React.ReactNode;
+  /**
+   * Whether to show the last separator
+   */
+  showLastSeparator?: boolean;
+};
+```
+
+### BreadcrumbItem
+
+```ts
+type BreadcrumbItemProps = {
+  /**
+   * Href of the BreadcrumbItem
+   */
+  href: string;
```

Comment:
Its always clickable and link right? never a text or button

Reply:
Yes, a clickable link. <a \/>